src/lib/plugin_apis/crypto.api

```diff
@@ -309,6 +309,27 @@ gboolean bd_crypto_luks_resize (const gchar *luks_device, guint64 size, GError *
  */
 gboolean bd_crypto_tc_open (const gchar *device, const gchar *name, const guint8* pass_data, gsize data_len, gboolean read_only, GError **error);
 
+/**
+ * bd_crypto_tc_open2:
```

Comment:
I suggest ``bd_crypto_tc_open_full`` as a better name for this function.

Reply:
Done in 94aca5c

Reply:
I squashed commit 94aca5c into 9a7f054 during the rebasing.